blog/_posts/2017-08-02-bear-hugs-from-200ok.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Bear Hugs from 200OK
+layout: post
+created_at: Wed Aug 02 2017
+permalink: blog/2017-08-02-bear-hugs-from-200ok
+current: blog
+author: Prachi & Ipshita
+twitter: Team200OK
+categories:
+- student-posts
+---
+
+Hello! We are Ipshita and Prachi of Team 200 OK, from New Delhi, India, working on coala this summer. One month into Rails Girls Summer of Code already and we didn’t even realise how time flew by. What a whirlwind of a month!
+
+### Two (Broke) Girls
+
+#### Fact Files : [Ipshita](https://github.com/ipshitac)
+- She is extremely meticulous and will not take anything up without making a checklist for it
+- She has a very eclectic taste in ‘get psyched’ music, much of which blares through her headphones as she works through this summer. Also, she loves to sing as she works, much to the annoyance of her teammate (sorry Prachi!)
+- If you want her to be your friend, just get her a cup of coffee and something nice to eat. She is a huge foodie!
+- She can stop almost any work midway and scribble poetry at the first piece of paper she gets her hands on.
+- She is already overjoyed that you’re reading about her! _blushes_
+
+#### Fact Files : [Prachi](https://github.com/prachi1210) 
+- She is the antithesis of Ipshita. Her desktop and working area is disorganised chaos. You can only and only find your way through folders, if she is there to guide you.  
+- She works well under pressure, but only if she has a deadline to meet. 
+- There is no easy way to say this but her jokes suck.
+- She reads. A lot. Like a person who has been starved of any reading material for years. She also quotes random facts that have no relation whatsoever with the conversation on hand.
+- She is secretly dancing at the prospect of people reading this and her becoming famous. _(You have high hopes, Prachi)_
+
+<p align="center">
```

Comment:
could you remove the `<p align="center">` & `</p>` tags around this image? There is styling already applied to all blog images by default in our css stylesheet, so you don't need to center this separately :)